src/index.jsx

```diff
@@ -278,8 +331,10 @@ CalendarHeatmap.defaultProps = {
   gutterSize: 1,
   horizontal: true,
   showMonthLabels: true,
+  showWeekdayLabels: true,
```

Comment:
should this be `false` by default?

Reply:
Sure, I shouldn't change the old behaviour here.

Reply:
I've updated PR with this change